### PR TITLE
Feature/document passage UI

### PR DESCRIPTION
**Pull request by PatFawbertMills**
- Highlighting the selected passage next to the PDF preview
- Comment out all unused annotation code in PDF renderer

### CODE REVIEW

File: frontend/src/components/EmbeddedPDF.tsx [hunk 1]
```diff
@@ -17,90 +17,90 @@ const EmbeddedPDF = ({ document, passageIndex = null }: TProps) => {
   const { createPDFClient, passageIndexChangeHandler } = useMemo(() => usePDFPreview(document), [document]);
 
   // TODO: refactor and enable annotation highlighting
-  const annotationManagerConfig = {
-    showToolbar: false,
-    showCommentsPanel: false,
-    downloadWithAnnotations: true,
-    printWithAnnotations: true,
-  };
+  // const annotationManagerConfig = {
```
Comment (diversemix) [inline frontend/src/components/EmbeddedPDF.tsx:20]: Do we need this commented out code still in the file?

File: frontend/src/components/PassageMatches.tsx [hunk 2]
```diff
@@ -5,21 +5,22 @@ type TProps = {
...
           <Loader />
         </div>
       ) : (
-        <div className="px-6">
+        <>
           {document.document_passage_matches.map((item, index: number) => (
             <div
               key={item.text_block_id}
-              className="py-4 cursor-pointer"
+              className={`p-4 cursor-pointer border hover:bg-offwhite ${activeIndex === index ? "border-blue-200 bg-lightgray" : "border-transparent"}`}
```
Comment (diversemix) [inline frontend/src/components/PassageMatches.tsx:23]: might be worth naming these css classes in their own right e.g. `.document-index-active` / `document-index-inactive` ?

**Review by diversemix — APPROVED** [review]
lgtm - just a couple of comments/suggestions

File: frontend/src/components/EmbeddedPDF.tsx (hunk 1 above)
Comment (PatFawbertMills) [reply frontend/src/components/EmbeddedPDF.tsx:20]: It is related to the passage highlighting work that Paula started, I'll pull it out and store it separately.

**Pull request merged**